.env

```diff
@@ -1,2 +1,3 @@
 PORT = 8080
```

Comment:
Please remove this file! This is a necessity now on. Your PR will not be merged if this file is in it!